tests/Streamer.php

```diff
@@ -71,6 +71,84 @@ public function providesNameAndData()
 		];
 	}
 
+	/**
+	 * @return array array(filename, mimetype), expectedMimetype, expectedFilename, $description, $browser
+	 */
+	public function providerSendHeadersOK() {
+		return array(
+			// Regular browsers
+				array(
+						array(),
+						'application/x-tar',
+						'archive.tar',
+						'default headers',
+						'Mozilla/5.0 (Windows NT 6.1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/41.0.2228.0 Safari/537.36',
+						'Content-Disposition: attachment; filename*=UTF-8\'\'archive.tar; filename="archive.tar"',
+				),
+				array(
+						array(
+								'file.tar',
+								'application/octet-stream',
+						),
+						'application/octet-stream',
+						'file.tar',
+						'specific headers',
+						'Mozilla/5.0 (Windows NT 6.1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/41.0.2228.0 Safari/537.36',
+						'Content-Disposition: attachment; filename*=UTF-8\'\'file.tar; filename="file.tar"',
+				),
+			// Internet Explorer
+				array(
+						array(),
+						'application/x-tar',
+						'archive.tar',
+						'default headers',
+						'Mozilla/5.0 (compatible, MSIE 11, Windows NT 6.3; Trident/7.0; rv:11.0) like Gecko',
+						'Content-Disposition: attachment; filename="archive.tar"',
+				),
+				array(
+						array(
+								'file.tar',
+								'application/octet-stream',
+						),
+						'application/octet-stream',
+						'file.tar',
+						'specific headers',
+						'Mozilla/5.0 (compatible, MSIE 11, Windows NT 6.3; Trident/7.0; rv:11.0) like Gecko',
+						'Content-Disposition: attachment; filename="file.tar"',
+				),
+		);
+	}
+
+	/**
+	 * @dataProvider providerSendHeadersOK
+	 * @preserveGlobalState disabled
+	 * @runInSeparateProcess
+	 *
+	 * @param array $arguments
+	 * @param string $expectedMimetype
+	 * @param string $expectedFilename
+	 * @param string $description
+	 * @param string $browser
+	 * @param string $expectedDisposition
+	 */
+	public function testSendHeadersOKWithRegularBrowser(array $arguments,
+														$expectedMimetype,
+														$expectedFilename,
+														$description,
+														$browser,
+														$expectedDisposition) {
+		$_SERVER['HTTP_USER_AGENT'] = $browser;
+		call_user_func_array(array($this->streamer, "sendHeaders"), $arguments);
+		$headers = xdebug_get_headers();
```

Comment:
this is breaking travis - most probably because xdebug is no enabled with php 7 on travis

@LukasReschke mind having a second look? THX